templates/spinup.sh.j2

```diff
@@ -107,12 +107,12 @@ network-interfaces: |
   auto eth0
   iface eth0 inet static
     address $4
-    network {{ system_network }}
-    netmask {{ system_netmask }}
-    broadcast {{ system_broadcast }}
-    gateway {{ system_gateway }}
-    dns-nameservers {{ system_nameservers }}
-    dns-search {{ system_dns_search }}
+    network {{ system_network | default(null) }}
```

Comment:
In this case, still "iface eth0 inet static" statement is remained and defined. 
We may need to avoid to create file 'ifcfg-eth0' in case of non-static ip address case.
```
iface eth0 inet static
  address 
  network null
   (snip)
...
```